src/components/fargekategori/bekreft-endre-fargekategori-pa-mange-modal.css

```
@@ -0,0 +1,5 @@
+.bekreft-fargekategori-pa-mange-modal .kategori-som-skal-bekreftes {
+    /* Gjer ikonet midstilt til teksten den står saman med, sjølv om teksten kjem på to linjer */
+    position: relative;
+    top: 0.45rem;
```

Skjønner ikke hvorfor SVGen gjør høyden til `p` større når de begge er `24px`??
Men for å unngå det så kan vi visst bruke:
```suggestion
    vertical-align: middle;
```
Da blir høyden til `p` igjen `24px` og den er justert på samme måte!